meteor-app/imports/ui/components/textmenu.js

```diff
@@ -0,0 +1,104 @@
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import MenuItem from 'material-ui/MenuItem';
+import FlatButton from 'material-ui/FlatButton';
+import Menu from 'material-ui/Menu';
+import Popover from 'material-ui/Popover';
+import Divider from 'material-ui/Divider';
+import React from 'react';
+import _ from 'lodash';
+
+export default class TextMenu extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { open: [] };
+    this.data = this.props.data;
+    this.style = this.props.style;
+    for (let i = 0; i < this.data.length; i += 1) {
+      this.state.open.push(false);
+    }
+    this.numberOfElements = 0;
```

Comment:
This variable is being incremented but is not reset anywhere. Is that intended behavior? Why do you need this to be continuously growing?